src/extensions/common/workspaceReducer.ts

```diff
@@ -0,0 +1,159 @@
+import { Recording, Sorting, SortingCuration } from "../extensionInterface"
```

Comment:
The workspace reducer defines all of the workspace actions: add/remove sortings/recordings, etc.